releasenotes/notes/fix-missing-lib_config-f2c0a6a40f525bff.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    internal: This fix resolves an issue regarding the remote config module where a payload missing the ``lib_config`` entry would crash the application due to an unhandled ``KeyError`` exception.
```

```suggestion
    internal: This fix resolves an error regarding the remote config module with payloads missing a ``lib_config`` entry
```

I don't think this issue was crashing apps, it was just leading to error logs and failed to apply RC? The RC layer should be catching the errors gracefully

---

Well, I don't see where we're catching the errors, I think it would crash at preload here:
https://github.com/datadog/dd-trace-py/blob/0dab62944cde24316ff4a3c5e253352bf0856537/ddtrace/bootstrap/preload.py#L79

---

hmm it wasn't crashing my app that had the error. It was just logged. I suspect the RC middleware that we have handles errors in the handler method.